service/cziti/dns-proxy.go

```diff
@@ -320,7 +317,12 @@ outer:
 		if dnsRetryInterval >= 10000 {
 			dnsRetryInterval = 10000
 		}
-		goto outer
+		goto GetUpstream
+	} else {
+		log.Debugf("Upstream DNS dials succeeded.")
```

Comment:
this log is effectively the same as the one on line 328 and thus, i think this can be removed